ironfish-cli/src/ui/ledger.ts

```diff
@@ -42,6 +42,7 @@ export async function ledger<TResult>({
     // eslint-disable-next-line no-constant-condition
     while (true) {
       try {
+        ux.action.status = undefined
```

Comment:
I thought about this, but the only real issue here is that if the user is in an error loop this might cause it to keep flickering and make it hard to read right? The error will display for 1 second and then be deleted. Let's test this.